examples/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
     project(zenohpico_examples LANGUAGES C)
     include(../cmake/helpers.cmake)
     set_default_build_type(Release)
-    configure_include_project(ZENOHPICO zenohpico zenohpico::static ".." zenohpico "https://github.com/eclipse-zenoh/zenoh-pico" "")
+    configure_include_project(ZENOHPICO zenohpico zenohpico::lib ".." zenohpico "https://github.com/eclipse-zenoh/zenoh-pico" "")
```

Comment:
This should now work regardless of whether zenohpico is included using `find_package` or `add_subdirectory` as both define `zenohpico::lib` target